com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -922,6 +922,10 @@ public virtual void Heuristic(in ActionBuffers actionsOut)
         /// </summary>
         internal void InitializeSensors()
         {
+            if (m_PolicyFactory == null)
+            {
+                m_PolicyFactory = GetComponent<BehaviorParameters>();
```

Comment:
is this needed here?  It's called in LazyInitialize before InitializeSensors.

Reply:
I need it because I do not call LazyInitialize (or initialize) in the modelloaderchecks. I call Initialize Sensors directly.